.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,83 @@
+name: allcontributors-auto-detect
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Tributors Update      
+        uses: con/tributors@0.0.18
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # Single text list (space separated) of parsers, leave unset to auto-detect
+          parsers: unset
+
+          # Update lookup with GitHub metadata
+          update_lookup: github
+
+          # Skip these users (example)
+          skip_users: manbat
+
+          # INFO, DEBUG, ERROR, WARNING, etc.
+          log_level: DEBUG
+
+          # If files already exist and an init is done, force overwrite
+          force: true
+
+          # the minimum number of contributions required to add a user
+          threshold: 1
+
+      - name: Checkout New Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH_AGAINST: "maint"
```

Comment:
I don't see relevant discussion, so will start a new one: why here we are not picking up the target branch PR is intended for, or current branch this workflow runs for ATM if not a PR?  My point was: we have multiple branches against which people could submit a PR - e.g. it could be `maint` or `master`.  I guess `tributors` should operate on that branch, not some hard coded maint or master.

Reply:
It should be run after a contribution is merged (and thus final), which is reliably to master or maint.

Reply:
exactly - "master or maint", so it should not be hardcoded to maint here -- it might arrive to master. isn't there some variable available already to use here?

Reply:
It should be master, actually, and a variable (if it were available) would be less clear to the reader/developer.